### PR TITLE
Add locale with sem-service

### PR DESCRIPTION
**Pull request by csidyel**
Info pulled from https://github.com/renderedtext/tasks/issues/493
Another customer asked for it

### CODE REVIEW

File: docs/faq/faq.md [hunk 1]
```diff
@@ -143,6 +143,36 @@ To get the job ID you can use <code>sem get jobs</code> to get the list of all r
...
+  
+1. Create a Dockerfile with the following:
+```
+FROM postgres:9.6
+RUN localedef -i ru_RU -c -f UTF-8 -A /usr/share/locale/locale.alias ru_RU.UTF-8
+ENV LANG ru_RU.UTF-8
+```
+2. Rebuild the Postgres image using the locale:
+```
+docker build - -t postgres:lang < Dockerfile
```
Comment (bogyo210) [inline docs/faq/faq.md:167]: here 'lang' is not literal, in this specific case it is 'ru'

File: docs/faq/faq.md [hunk 2]
```diff
@@ -143,6 +143,36 @@ To get the job ID you can use <code>sem get jobs</code> to get the list of all r
...
+```
+3. Start the newly created image:
+```
+docker run --rm --net host -d -e POSTGRES_PASSWORD=semaphore --name postgres -v /var/run/postgresql:/var/run/postgresql postgres:lang
```
Comment (bogyo210) [inline docs/faq/faq.md:171]: here 'lang' at the end should/could be 'ru' if we follow the example above

File: docs/faq/faq.md (hunk 1 above)
Comment (csidyel) [reply docs/faq/faq.md]: updated

File: docs/faq/faq.md (hunk 2 above)
Comment (csidyel) [reply docs/faq/faq.md]: updated

**Pull request merged**